src/VirtualClient/VirtualClient.Dependencies/AMDGPUDriverInstallation.cs

```diff
@@ -408,5 +410,18 @@ private async Task InstallAMDGPUDriverOnV620(string driverPackagePath, EventCont
                 process.ThrowIfErrored<DependencyException>(ProcessProxy.DefaultSuccessCodes, errorReason: ErrorReason.DependencyInstallationFailed);
             }
         }
+
+        private async Task ReportAMDDriverVersionWindows(EventContext telemetryContext, CancellationToken cancellationToken)

```

Comment:
Could you make this a best effort? I believe this will crash VC if the process fails.